recipes/kmc-smudge-pairs/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "kmc_smudge_pairs" %}
+{% set version = "3.1.2rc4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gkanogiannis/KMC/archive/v{{ version }}.tar.gz
+  sha256: 43ae55aabc1334a038e5898b2fdc9a1d254d6a434c9ad34e56663be1aec9dc42
+
+build:
+  skip: true  # [not linux]
+  number: 0
+  run_exports:
+    - zlib
+    - bzip2
```

Comment:
This is in the packages themselves and should not be here.
```suggestion
```

Reply:
This was removed.